### DISCUSSION TITLE
questions about software usage

### DISCUSSION THREAD

**Discussion by beichu** [blueseerERP/blueseer#76] [Q&A] [answered]
Hello, I have a couple of questions regarding how to use blueseer:

1. Will the software still function if I don't use departments/cc?
2. Are there depreciation and amortization schedule functionalities?

Thanks!

**Comment by vaughnte** [comment, marked as answer]
1.  You will have to use at least 1 dept/cc.   You can call it 9999, 8888, etc....with description 'generic' if you wish.   There is no adverse behaviour for using a generic dept/cc.....but the coding requires at least 1 defined.
2.   Currently, there are no depreciation or amortization mechanisms within the financial menus.   A very good prospect for future enhancement though.

**Reply by beichu** [reply]
Got it. Thank you!
